### PR TITLE
Default Unit

### LINKED ISSUE #8: Default unit
Curious if anyone is using a mixture of `TimeUnit`s? Was thinking about changing the api to have a default unit of milliseconds so it's a little cleaner:

```elm
Delay.after 500 SomethingTriggered
```

```elm
Delay.sequence
    [ ( 100, MsgOne )
    , ( 200, MsgTwo )
    , ( 100, MsgThree )
    ]
```

And maybe add some helpers to convert millis to other time units:

```elm
Delay.after (Delay.seconds 5) SomethingTriggered
```

```elm
seconds : Millis -> Millis
seconds millis =
    1000 * millis
```

@lenards given you've been using this recently do you have any thoughts?

### PR DESCRIPTION
**Pull request by andrewMacmurray**
+ Updates api to have default millisecond unit
+ CI builds docs and examples

Resolves #8

### CODE REVIEW

**Comment by andrewMacmurray** [comment]
@lenards no rush or pressure but would be awesome if you could take a look at this one

File: package.json [hunk 1]
```diff
@@ -9,14 +9,18 @@
   "devDependencies": {
     "ava": "^3.15.0",
     "elm": "^0.19.1-5",
+    "elm-doc-preview": "^5.0.5",
```
Comment (lenards) [inline package.json:12]: I hadn't checked this library out until recently. It's pretty cool.

File: src/Delay.elm [hunk 2]
```diff
@@ -116,16 +92,37 @@ addOffset previousTotal time unit =
...
+minutes : Millis -> Millis
+minutes millis =
+    seconds millis * 60
+
+
+{-| Convert `milliseconds` to `hours`
+-}
+hours : Millis -> Millis
+hours millis =
+    minutes millis * 60
```
Comment (lenards) [inline src/Delay.elm:113]: I think this works out really, really well.

**Review by lenards — APPROVED** [review]
I think this works out really well! 🎉 
It's a good move to simplify 🙇 

I was tempted to think of a name to _alias_ `(Millis -> Millis)` functions to ... but that likely obscures what they are (and would make it more difficult to see that I could define my own - if I wanted). So I - perhaps - should I have omitted that 😉. Just sharing on the off-chance you had a similar thought.

File: package.json (hunk 1 above)
Comment (andrewMacmurray) [reply package.json:12]: I only discovered it recently, it's a really nice feedback loop when you're working on a library!

**Comment by andrewMacmurray** [comment]
That's a good idea for aliasing  `Millis -> Millis`, maybe `ToUnit`?

**Comment by lenards** [comment]
That’s a perfect name! 🙌

**Comment by andrewMacmurray** [comment]
Gonna merge this in now, thanks for your help @lenards!

**Pull request merged**